docs/source/fx.rst

```diff
@@ -293,10 +293,188 @@ Writing Transformations
 
 TODO
 
-Debugging Transformations
--------------------------
+Debugging
+-----------
+
+Introduction
+^^^^^^^^^^^^^^^^
+
+Once we've generated Python code from our symbolically-traced Module,
+we'll want to verify that the proper semantics were preserved. If they
+weren't, we may need to do some debugging. The key is to work
+backwards: first, check the results of the generated module, then debug
+the generated code, then debug the process of transformations that lead
+to the generated code.
+
+If you’re not familiar with debuggers, please see the auxiliary section
+:ref:`Available Debuggers`.
```

Comment:
This reference link doesn't work, I think you'll need to add an anchor tag like here: https://github.com/pytorch/pytorch/pull/50638/files#diff-8cd3ad665631e6f4e42aed8aa144620de5a93efa7e0388094ff3485c3334e0faR195
and refer to that